# Message header
Header header

# Defines the safety zone circle radius (in meters) by safety level
# - clear_zone is the most external to the robot
# - warning_zone is the intermediary zone
# - critical_zone is the closest to the robot

float64 clear_zone_radius
float64 warning_zone_radius
float64 critical_zone_radius
